src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1412,10 +1414,21 @@ treatAsInline = Set.fromList
   , "pagebreak"
   ]
 
+dolabel :: PandocMonad m => LP m Inlines
+dolabel = do
+  v <- braced
+  return $ spanWith ("",[],[("data-label", toksToString v)]) $ inBrackets $ str $ toksToString v
```

Comment:
I just changed `setCaption` to use "label", so please rebase against the latest, and go ahead and use `label` instead of `data-label` here as well.